test/MultiCompiler.test.js

```diff
@@ -397,6 +397,59 @@ describe("MultiCompiler", () => {
 				env.callback.callCount.should.be.exactly(1);
 			});
 		});
+
+		describe("with missing compiler dependencies", () => {
+			beforeEach(() => {
+				setupTwoCompilerEnvironment(env, {
+					name: "compiler1",
+					dependencies: ["compiler2"]
+				}, {
+					name: "compiler3"
+				});
+				env.callback = sinon.spy();
+				env.options = [{
+					testWatchOptions: true
+				}, {
+					testWatchOptions2: true
+				}];
+				env.result = env.myMultiCompiler.watch(env.options, env.callback);
+			});
+
+			it("should call the callback with an error message", () => {
+				env.compiler1WatchCallbacks.length.should.be.exactly(0);
+				env.compiler2WatchCallbacks.length.should.be.exactly(0);
+				env.callback.callCount.should.be.exactly(1);
+				env.callback.getCall(0).args[0].should.be.Error();
+				should(env.callback.getCall(0).args[1]).be.undefined();
+			});
+		});
+
```

Comment:
Could you add a test with this configuration?

``` js
{
  name: "a",
},
{
  name: "b",
  dependencies: ["a", "c"]
},
{
  name: "c",
  dependencies: ["b"]
}
```

This is a test for `if(!edges.find(e => e.target === target))`